touchforms/backend/xformplayer.py

```diff
@@ -529,7 +529,7 @@ def delete_repeat (self, rep_ix, _junc_ix=None):
         if (junc_ix):
             self.fec.jumpToIndex(junc_ix)
 
-        self.fec.deleteRepeat(rep_ix - 1)
```

Comment:
@benrudolph Looks like this might be used for deleting a repeat for a repeat group question?  If so, there's not a way to delete a repeat from smsforms right now so it wouldn't be impacted if that's the case.